src/main/java/org/opensearch/security/securityconf/impl/CType.java

```diff
@@ -50,21 +51,24 @@
 
 public enum CType {
 
-    INTERNALUSERS(toMap(1, InternalUserV6.class, 2, InternalUserV7.class)),
-    ACTIONGROUPS(toMap(0, List.class, 1, ActionGroupsV6.class, 2, ActionGroupsV7.class)),
-    CONFIG(toMap(1, ConfigV6.class, 2, ConfigV7.class)),
-    ROLES(toMap(1, RoleV6.class, 2, RoleV7.class)),
-    ROLESMAPPING(toMap(1, RoleMappingsV6.class, 2, RoleMappingsV7.class)),
-    TENANTS(toMap(2, TenantV7.class)),
-    NODESDN(toMap(1, NodesDn.class, 2, NodesDn.class)),
-    WHITELIST(toMap(1, WhitelistingSettings.class, 2, WhitelistingSettings.class)),
-    ALLOWLIST(toMap(1, AllowlistingSettings.class, 2, AllowlistingSettings.class)),
-    AUDIT(toMap(1, AuditConfig.class, 2, AuditConfig.class));
+    ACTIONGROUPS(toMap(0, List.class, 1, ActionGroupsV6.class, 2, ActionGroupsV7.class), "action_groups.yml"),
```

Comment:
I like this - a same it wasn't in alpha order to begin with

Reply:
:-)